src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -27,7 +41,9 @@ export interface VideoThumbnailProps {
 
 export function VideoThumbnail({
   thumbnailUrl,
-  videoLength,
+  videoLength = 0,
+  videoProgress = 0,
+  showVideoProgress = true,
```

Comment:
Unless I'm missing something, this should default to `false`, and be set explicitly by the user if they want to show it.

```suggestion
  showVideoProgress = false,
```